client/packages/requisitions/src/RequestRequisition/ListView/ProgramRequisitionOptions.tsx

```diff
@@ -96,14 +116,14 @@ const LabelAndOptions = <T,>({
   value,
   autoFocus,
   optionKey,
-}: Pick<
-  AutocompleteProps<T>,
-  'options' | 'value' | 'optionKey' | 'autoFocus' | 'disabled'
-> & {
-  label: string;
-  set: (value: T | null) => void;
-  labelNoOptions?: string;
-}) => {
+  renderOption,
+  getOptionDisabled,
+}: Pick<AutocompleteProps<T>, 'optionKey' | 'autoFocus'> &
+  Common<T> & {
+    label: string;
+    set: (value: T | null) => void;
+    labelNoOptions?: string;
+  }) => {
```

Comment:
`Common` already includes label, set & labelNoOptions

```suggestion
  Common<T>) => {
```

Reply:
Ofcourse 🙏